### ISSUE TITLE
Feature Request: Allow the Ability to select and manipulate more than one ledger.

### ISSUE THREAD

**Issue by josarsepi** [Vifon/ledger-web#1]
It'd be great if a user could select (maybe from a drop down menu) a file to apply changes to.

This would allow for the include statement to be used in a master ledger file while all the transactions live in thier respective account ledger files.

**Comment by vifon** [comment]
So essentially a user would have multiple ledger journal files but only one "active" at a time? I'll try to add it when I get some time.

**Comment by josarsepi** [comment]
Yes.  I've been looking over the code myself but I'm not very familiar with Django so I'm certain it will take me much longer to make such a change.  Thanks for the follow up and this project.  It's fantastic.

**Comment by vifon** [comment]
I cannot give you a timeline but I'll keep it in mind when I'll get some free time.

Thanks for the kind words!
